build/gen_stub.php

```diff
@@ -854,6 +936,57 @@ function parseFunctionLike(
     );
 }
 
+function parseProperty(
+    Name $class,
+    PrettyPrinterAbstract $prettyPrinter,
+    int $flags,
+    Stmt\PropertyProperty $property,
+    ?Node $type
+): PropertyInfo {
+    $comment = $property->getDocComment();
+    $isDeprecated = false;
+    $isKnownName = false;
+    $docType = false;
+
+    if ($comment) {
+        $tags = parseDocComment($comment);
+        foreach ($tags as $tag) {
+            if ($tag->name === 'deprecated') {
+                $isDeprecated = true;
+            } else if ($tag->name === 'known') {
+                $isKnownName = true;
+            } else if ($tag->name === 'var') {
+                $docType = true;
+            }
+        }
+    }
+
+    $propertyType = $type ? Type::fromNode($type) : null;
+    if ($propertyType === null && !$docType) {
+        //throw new Exception("Missing type for property $class::\$$property->name");
```

Comment:
Somehow the phpdoc-based type hint cannot be found. It's because `$comment` is always `null`, so `$docType` is always `false`. Did I miss something?

Reply:
The doc comment will be on the property group (Stmt\Property).

Reply:
Thank you!